ViewControllers/DetailViewController.swift

```diff
@@ -12,17 +12,18 @@ class DetailViewController: UIViewController {
     static var identifier: String{NSStringFromClass(self)}
```

Comment:
👀
This is super nice! In case you wanna read more discussions around how to get a name from a type check this [thread](https://forums.swift.org/t/relying-on-string-describing-to-get-the-name-of-a-type/16391). Long story short, the piece of code you are using is the recommended one